lib/src/combine_latest.dart

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+
+/// Combine the latest value from the source stream with the latest value from
+/// [combineWith] using [combine].
+///
+/// No event will be emitted from the result stream until both the source stream
+/// and [combineWith] have each emitted at least one event. The result stream
+/// will not close until both the source stream and [combineWith] have closed.
+/// Errors throw by [combine], along with any errors on the source stream or
+/// [combineWith], are forwarded to the result stream.
+///
+/// If the source stream is a broadcast stream, the result stream will be as
+/// well, regardless of [combineWith]'s type. If a single subscription stream is
+/// merged into a broadcast stream it may never be canceled.
```

Comment:
nit: "merge" is a different operator, maybe "a single subscription stream is combined with"

Reply:
Done